src/AttributeGroup.php

```diff
@@ -0,0 +1,41 @@
+<?php
+namespace SnowIO\AkeneoDataModel;
+
+class AttributeGroup
+{
+    public function getCode(): string
+    {
+        return $this->code;
+    }
+
+    public function getLabels(): array
+    {
+        return $this->labels;
+    }
+
+    public function getLabel(string $locale): string
```

Comment:
The return type here is wrong. Please check for the same mistake in other classes.